homeassistant/core.py

```diff
@@ -250,16 +265,6 @@ def async_run_job(self, target: Callable[..., None], *args: Any) -> None:
         else:
             self.async_add_job(target, *args)
 
-    def _loop_empty(self) -> bool:
```

Comment:
The reason the tests are failing is probably the removal of this method. However, I don't even get how it is working currently because we actually yield from `async_block_till_done` which yields from `run_in_executor` to call this method 🤔